metrics/README.md

```diff
@@ -0,0 +1,15 @@
+# Metrics Tools
+
+## get_prs.py
+
+This tool gets the PR counts for 2022 and stores them in `prs.csv`, if it is interrupted the current week number fetched is stored in `prs_start.txt` so it can continue. This also means that it can be run as a script and will only append as required when there is new data. There is a deliberate 2 second pause between each request so as not to hit the GitHub rate limit (30 requests per minute).
```

Comment:
can we give the year/month as argument? I think it would be better if the year is not hardcoded in the script.